source/_data/themes.yml

```diff
@@ -437,14 +437,16 @@
     - light
     - colorful
 - name: Inside
-  description: Flat and clean theme ❤️
+  description: ❤️ SPA, flat and clean theme for Hexo.
```

Comment:
We homogenized description in #899 
Could you please delete `for Hexo`?

Reply:
```suggestion
  description: ❤️ SPA, flat and clean theme.
```